frontend/mobilityai/app/src/main/java/com/ai/mobility/mobilityai/MainActivity.java

```diff
@@ -92,9 +77,13 @@
     private ArrayList<BluetoothDevice> m_devices;
     private static final long SCAN_PERIOD = 10000L;
 
+    final Calendar cal = Calendar.getInstance();
 
     //TODO: Remove once the server side changes are made
```

Comment:
What server side changes?

Reply:
The Endpoints need to be updated to properly get/set users. I'll add those once we do the rework on the api.